pages/chat/_entity.vue

```diff
@@ -23,25 +23,26 @@
 
           <bottom-sheet :items="bottomSheetItems" ref="mobileDialog"/>
         </template>
-
-        </div>
+      </div>
     </header-bar>
 
     <!-- Main Section -->
     <main class="d-flex flex-row flex-grow-1 hide-overflow">
-      <div class="d-flex flex-column flex-grow-1 os-host-flexbox">
+      <div class="d-flex flex-column flex-grow-1">
         <!-- Message List -->
-        <overlay-scrollbars
+        <simplebar
+          class="simplebar wide-scrollbar flex-grow-1" 
+          data-simplebar-auto-hide="false" 
+          data-simplebar-force-visible="true"
           ref="messageList"
-          class="flex-grow-1 flex-shrink-1 wide-scrollbar"
```

Comment:
Didn't `.wide-scrollbar` also draw a slightly transparent (but visible) track, too? I don't think `<simplebar/>` has that by default.

Reply:
Added.